graphql_compiler/query_pagination/parameter_generator.py

```diff
@@ -169,7 +169,7 @@ def _compute_parameters_for_non_uuid_field(
     #                       that are known to be empty. This can cause the number of generated
     #                       pages to be less than the desired number of pages.
     return _deduplicate_sorted_generator(
-        proper_quantiles[index]
+        relevant_quantiles[index]
```

Comment:
Bug wasn't caught before since we didn't have a test paginating recursively on a non-uuid field